pkg/gadgettracermanager/match/match.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package match
+
+import (
+	pb "github.com/kinvolk/inspektor-gadget/pkg/gadgettracermanager/api"
+)
+
+func ContainerSelectorMatches(s *pb.ContainerSelector, c *pb.ContainerDefinition) bool {
```

Comment:
I moved it together with `pkg/container-collection` instead. I think it's better to avoid `containerutils` for this because containerutils contains generic low-level helpers independent of Inspektor Gadget and it is imported in an external project. Arguably, `containerutils` should be a top-level package and not part of the `gadgettracermanager` directory.

And `ContainerSelectorMatches()` is specific to Inspektor Gadget (with IG's data structures).